src/Benchmarks/Middleware/JsonMiddleware.cs

```diff
@@ -10,7 +10,7 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 
-#if NETCOREAPP3_0 || NETCOREAPP3_1 || NETCOREAPP5_0 || NET5_0 || NET6_0_OR_GREATER
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
@sebastienros ?

Reply:
I changed other benchmarks recently to target 6 at mininum. I think we can do the same here as we haven't had to run it even for 5 since last year.

The excuse to not do it earlier is that we always talk about getting rid of this project and refactor it in more granular apps (middleware, mvc) and updating it to the latest templates. That some good amount of work because we will need to update the benchmarks configurations and CI too. But this could be done in multiple steps.